src/pools/fixedRate/FixedRateExchange.ts

```diff
@@ -128,14 +130,22 @@ export class FixedRateExchange {
     datatokenAddress: string,
     dtAmount: string,
     maxBaseTokenAmount: string,
+    consumeMarketAddress: string,
+    consumeMarketFee: string,
     contractInstance?: Contract
   ): Promise<number> {
     const fixedRate = contractInstance || this.fixedRateContract
     const gasLimitDefault = this.GASLIMIT_DEFAULT
     let estGas
     try {
       estGas = await fixedRate.methods
-        .buyDT(datatokenAddress, dtAmount.toString(), maxBaseTokenAmount.toString())
+        .buyDT(
+          datatokenAddress,
+          dtAmount.toString(),
```

Comment:
either the typing is wrong (says `dtAmount` comes in as a `string`) or this `.toString()` is wrong